tests/kohana/Log/FileTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * Tests Kohana Logging API - writing to file
+ *
+ * @group kohana
+ * @group kohana.core
+ * @group kohana.core.logging
+ * @group kohana.core.logging.file
+ *
+ * @package    Kohana
+ * @category   Tests
+ * @author     Kohana Team
+ * @copyright  (c) 2008-2012 Kohana Team
+ * @license    http://kohanaframework.org/license
+ */
+class Kohana_Log_FileTest extends Unittest_TestCase {
+
+	private $log_dir;
+
+	/**
+	 * Sets up the test enviroment
+	 */
+	// @codingStandardsIgnoreStart
+	function setUp()
+	// @codingStandardsIgnoreEnd
+	{
+		parent::setUp();
+
+		$this->log_dir = sys_get_temp_dir()
```

Comment:
I think you should remove this directory in tearDown - or better yet use mikey179/vfsStream to test the file writer against a virtual filesystem.